image_processing/requirements.txt

```diff
@@ -0,0 +1,37 @@
+bokeh==2.3.0
+certifi==2020.12.5
+click==7.1.2
+cycler==0.10.0
+decorator==4.4.2
+h11==0.12.0
+h2o-wave==0.12.1
+httpcore==0.12.3
+httpx==0.16.1
+idna==3.1
+imageio==2.9.0
+Jinja2==2.11.3
+kiwisolver==1.3.1
+MarkupSafe==1.1.1
+matplotlib==3.3.4
+networkx==2.5
+numpy==1.20.1
+opencv-python==4.5.1.48
+packaging==20.9
+pandas==1.2.3
+Pillow==8.1.0
+pkg-resources==0.0.0
```

Comment:
When I run `make setup` on OS X I get an error for pkg-resources==0.0.0, it looks like this might get added when you run pip freeze but shouldn't be in the requirements.txt I think. After removing this the app installed perfectly